.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        python-version: ['3.7', '3.8', '3.9', '3.10']
+        python-version: ['3.8', '3.9', '3.10', '3.11']
```

Comment:
@takojunior  it seems "windows-latest" is failing installing a (some?) library. That's not sth we can fix easily assume. In this case, I am OK dropping windows testing. That's in line our jump to v2.0 where we drop support for Python 3.7 "and" "windows". May be remove windows and merge? Please add windows change to CHANGELOG as well

Reply:
Makes sense to me. I see the updates to CHANGELOG. I will update PyPI.